src/doc/documentation-guide.mk2

```diff
@@ -0,0 +1,189 @@
+# Contributing Documentation in SnabbSwitch
+
+*XXX - TODO: Document different documentation targets: Casual users,
+ power users, contributors.*
```

Comment:
Meta-answer: I would say that each `program` can choose individually how it wants to present itself to users. For example, `snsh` will be very Lua-scripting-oriented but `packetblaster` might prefer to blend in with the usual Unix/C tools.

The decision is a little tricky in programs like `snabbnfv` where Lua is convenient but possibly distracting. I suppose the documentation guide does not have to tell people what to do in these situations beyond suggesting being thoughtful. If some kind of common practice arises over time then we could highlight that.